index.html

```diff
@@ -67,7 +68,219 @@ <h1 class="header-text">
+            </h1>
+            <div class="sub">
+                <p>Multi-Post
+                    Stories</p>
+                <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+
+            <div class="article">
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <section class="built-with">
+
+                    <button id="l" type="submit">
+                        html
+                    </button>
+                    <button id="o" type="submit">
+                        css
+                    </button>
+                    <button id="r" type="submit">
+                        javaScript
+                    </button>
+                </section>
+
+                <section class="prompt">
+                    <button type="submit">See Project</button>
+                </section>
+            </div>
+        </div>
+    </div>
+    <div class="about">
+        <h1 class="header-text">
+            About Myself
+        </h1>
+
+        <p class="text">
+            Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+            my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.
+        </p>
+        <p class="connect">
+            LET’S CONNECT
+        </p>
+        <ul class="socialmedia">
+            <li><img src="./images/git.svg" alt="git"> </li>
+            <li> <img src="./images/linkedin.svg" alt="linkedin"> </li>
+            <li> <img src="./images/tumblur.svg" alt="tumblur"> </li>
+            <li> <img src="./images/twitter.svg" alt="twitter"> </li>
+            <li><img src="./images/medium.svg" alt="medium"> </li>
+        </ul>
+        <section class="prompt">
+            <button type="submit">Get my resume</button>
+        </section>
+        <div class="languages">
+            <section class="lang-drop">
+                <ul>
+                    <li>
+                        <div class="lang-head">
+                            <p>Languages</p>
+                            <img src="./images/down.png" alt="dropdown">
+                        </div>
+                        <div id="buttons">
+                            <button type="button">
+                                <img src="./images/js.svg" alt="js">
+                                javaScript
+
+                            </button>
+                            <button type="button">
+                                <img src="./images/html.svg" alt="html">
+                                HTMl
+                            </button>
+                            <button type="button">
+                                <img src="./images/css.svg" alt="css">
+                                css
+                            </button>
```

Comment:
- [ ] Please make sure the languages are list items by using the li  tag instead of button
- [ ] Also ensure the spell the languages correctly e.g html, css, javascript